Wrappers/Python/ccpi/viewer/CILViewer.py

```diff
@@ -604,21 +609,16 @@ def installPipeline(self):
 
         self.adjustCamera()
 
-        self.saveDefaultCamera()
-
         self.iren.Initialize()
         self.renWin.Render()
 
     def saveDefaultCamera(self):
-        camera = vtk.vtkCamera()
-        camera.SetFocalPoint(self.getCamera().GetFocalPoint())
-        camera.SetPosition(self.getCamera().GetPosition())
-        camera.SetViewUp(self.getCamera().GetViewUp())
-        self.default_camera = camera
+        self.default_camera = CameraData(self.getCamera())
 
     def resetCameraToDefault(self):
         if hasattr(self, 'default_camera'):
-            self.ren.SetActiveCamera(self.default_camera)
+            self.adjustCamera(resetcamera=True)
+            self.default_camera.copy_data_to_other_camera(self.getCamera())
```

Comment:
I'd drop `other` because there's no other camera in this call.
Also `copy_data` seems to be generic. I think this is also very confusing as the data that is copied is not quite explicit.

Why not making a `staticmethod` in `CameraData` along these lines

```python

class CameraData(object):

    @staticmethod
    def CopyDataToCamera(camera_data, vtkcamera):
        ...

```

This would then be used as 
```python

CameraData.CopyDataToCamera(self.default_camera, self.getCamera())
```

Reply:
sounds good

Reply:
I'll also rename self.default_camera to self.default_camera_data